src/expression/types/formatted.ts

```diff
@@ -1,19 +1,24 @@
 import type {Color} from '../../expression/types/color';
 import type {ResolvedImage} from '../types/resolved_image';
 
+export const VERTICAL_ALIGN_OPTIONS = ['bottom', 'center', 'top'] as const;
```

Comment:
Shouldn't this be defined in the v8 spec file and not here?

Reply:
I don't see a way how to do it - there is no other expression where this kind of definition lives in `v8.json` (or I can't find it). For `interpolate` expression type definition lives in the code (https://github.com/maplibre/maplibre-style-spec/blob/93a8cd30b84b815ea57d49172959038a57a86759/src/expression/definitions/interpolate.ts#L17).